src/amo/components/PermissionsCard/styles.scss

```diff
@@ -0,0 +1,29 @@
+.PermissionCard-learn-more {
+  margin: 12px auto 0;
+  width: 100%;
+
+  .Icon {
+    margin-left: 10px;
+  }
+}
+
+.PermissionsCard-subhead {
+  color: #737373;
+}
+
+.PermissionsCard-list {
+  list-style: none;
+  margin: 10px 0 0;
+  padding: 0;
+
+  li {
+    display: block;
+    margin-left: 10px;
+    margin-top: 10px;
+    vertical-align: middle;
```

Comment:
This only works for table cells. It's better to use flexbox as I suggested up above.